posthog/urls.py

```diff
@@ -94,7 +94,15 @@ def signup_to_team_view(request, token):
         team.users.add(user)
         team.save()
         posthoganalytics.capture(user.distinct_id, "user signed up", properties={"is_first_user": False})
-        posthoganalytics.identify(user.distinct_id, {"email_opt_in": user.email_opt_in})
+        posthoganalytics.identify(
+            user.distinct_id,
+            {
+                "email": request.user.email if not request.user.anonymize_data else None,
+                "company_name": team.name,
```

Comment:
If the user requested to anonymize data, shouldn't we also anonymize the company name?

Reply:
Looking more closely I see that perhaps this isn't to be changed here - But I'm wondering if we should collect company name if the admin has set anonymize data

Reply:
Well, I would argue that the company name isn't really PII, but we can definitely stop collecting / obfuscate the company name if the user chooses to anonymize the data, I added that to a separate discussion on #1497, I believe we could merge this in the meantime